docs/reference/objects/product/extra.md

```diff
@@ -4,52 +4,122 @@ title: Extra
 parent: Product
 ---
 
-When using a `extra` object you have access to the following attributes
+When using a `extra` object you have access to the following attributes.
 
-# extra.deposit_amount
+# extra.address
 
-Returns a humanized price for the extras deposit amount, if deposit's are not set on the extra this will just return `extra.price` e.g. `$250 Per Person`
+Returns the [address]({% link docs/reference/objects/product/address.md %}) associated with this extra.
+
+# extra.country
+
+Returns the name of the country for the product that this extra is part of.
+
+# extra.deposit_amount _(deprecated)_
+
+Returns a humanized price for the extras deposit amount, if deposit's are not set on the extra this will just return `extra.price` e.g. `$250 Per Person`.
+
+> **Deprecated**
+>
+> The deposit should be calculated in Liquid, using a mixture of the `extra.price` and `extra.product.deposit.rate`.
+>
+> {% raw %}
+> ```liquid
+> {% assign deposit_price = extra.price.fractional | times: extra.product.deposit.rate %}
+> {{deposit_price | money}}
+> ```
+> {% endraw %}
 
 # extra.has_infinite_stock
 
-Returns true if the extra has infinite stock
+Returns true if the extra has unlimited stock.
+
+# extra.hero_image
+
+Returns the hero [image]({% link docs/reference/objects/image.md %}) of the [product]({% link docs/reference/objects/product/index.md %}) this extra is belongs to.
 
 # extra.id
 
-Returns a unique id for the extra
+Returns a unique id for the extra.
 
 # extra.image
 
-Returns the extras [image]({% link docs/reference/objects/image.md %}) it it has one associated
+Returns the extras [image]({% link docs/reference/objects/image.md %}) it it has one associated.
```

Comment:
```suggestion
Returns the extras [image]({% link docs/reference/objects/image.md %}) if it has one associated, `nil` otherwise.
```

Reply:
Do you think it's necessary to include this? I kind of think it's assumed to returned nil unless "it has one associated".

Otherwise we might need to start adding "`nil` otherwise" to a bunch of other methods that potentially return nil, in order to be consistent.

Reply:
@ianmooney yeah maybe not. That was just a lil extra, I think we can just fix the typo and dismiss the rest.

Reply:
Ah yes, missed that 🙈 Thanks!